osu.Game/Screens/Multi/RealtimeMultiplayer/RealtimeMultiplayer.cs

```diff
@@ -57,7 +57,7 @@ protected override void UpdatePollingRate(bool isIdle)
 
         protected override Room CreateNewRoom()
         {
-            var room = base.CreateNewRoom();
+            var room = new Room { Name = { Value = $"{API.LocalUser}'s awesome room" } };
```

Comment:
Looks semi-awkward on the match creation screen with "match" everywhere but the textbox:

![2020-12-24-155901_892x252_scrot](https://user-images.githubusercontent.com/20418176/103095492-0227b380-4601-11eb-99c0-4bad94fdaf77.png)

Thoughts on

```suggestion
            var room = new Room { Name = { Value = $"{API.LocalUser}'s awesome match" } };
```

?

Reply:
sure. we're going to really need to standardise terms at some point.

Reply:
As discussed in discord, "realtime" has been reverted back to "room", while playlists have been changed to playlists.